IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -790,12 +822,13 @@ - (MSIDBaseToken *)getTokenWithAuthority:(MSIDAuthority *)authority
     return nil;
 }
 
-- (MSIDBaseToken *)getRefreshTokenByLegacyUserId:(NSString *)legacyUserId
-                                       authority:(MSIDAuthority *)authority
-                                        clientId:(NSString *)clientId
-                                        familyId:(NSString *)familyId
-                                         context:(id<MSIDRequestContext>)context
-                                           error:(NSError **)error
+- (MSIDBaseToken *)getRefreshableTokenByLegacyUserId:(NSString *)legacyUserId
```

Comment:
Instead of legacyUserId, should we call it displayableId?

Reply:
sure, changed :)